demos/meltdown_ac.cc

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * We create an array of words, shift them by one byte to make them unaligned
+ * and then we copy the public and private data into respective disaligned
+ * arrays - one character is stored into one unaligned word.
+ * Afterwards we turn on the alignment enforcement and try to read the
+ * disaligned private array with that enforcement. That always leads to SIGBUS
+ * however the unaligned data are processed speculatively.
+ * It is necessary to have the AM in CR0 register turned on, but on Linux it is
```

Comment:
expand terms like AM the first time you use them:
> AM ("alignment mask")

and it's worth mentioning what would happen if AM is _not_ set.

Reply:
Done.